.rspec

```diff
@@ -1 +1,2 @@
 --colour
+--require spec_helper
```

Comment:
@myronmarston thanks a lot for this explanation :heart: :heart: :heart: :heart: :heart: , I am really glad you are here to catch my mistakes :smile_cat: .

For now, I will remove this, and let the test files require spec_helper. After that we can try to split that in 2 files as you suggested.
thanks again.